src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -54,10 +87,27 @@ export const CreateTransaction = (props: CreateProps) => {
           onChange={(e) =>
             onInput({ ...formData, toAccountId: e.target.value })
           }
+          onBlur={fetchIfFederationAddress}
           value={formData.toAccountId}
           placeholder="Recipient's public key or federation address"
         ></TempInputEl>
       </El>
+      {federationAddressFetchStatus && (
+        <El>
+          {federationAddressFetchStatus === ActionStatus.PENDING && (
+            <El>Loading federation address...</El>
```

Comment:
`...` (three dots) is not the same as `…` (ellipsis). Where possible, you should use the real typographic character. See also https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#use-the-right-symbols